config/initializers/airbrake.rb

```diff
@@ -1,7 +1,17 @@
-if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
-  require 'airbrake'
+require 'airbrake'
 
+if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
   Airbrake.configure do |config|
     config.api_key = ENV['AIRBRAKE_API_KEY']
   end
+else
+  module Airbrake
```

Comment:
not a big fan of silent swallowing ... would an `attr_accessor :logger` fix it ?
a simple grep only shows the Airbrake.notify calls ... where are the others coming from ?